x/concentrated-liquidity/tick.go

```diff
@@ -30,6 +30,19 @@ func (k Keeper) initOrUpdateTick(ctx sdk.Context, poolId uint64, tickIndex int64
 	// calculate liquidityGross, which does not care about whether liquidityIn is positive or negative
 	liquidityBefore := tickInfo.LiquidityGross
 
+	// if given tickIndex is LTE to the current tick and the liquidityBefore is zero,
+	// set the tick's fee growth outside to the fee accumulator's value
+	if liquidityBefore.IsZero() {
+		if tickIndex <= currentTick {
+			accum, err := k.getFeeAccumulator(ctx, poolId)
+			if err != nil {
+				return err
+			}
+
+			tickInfo.FeeGrowthOutside = accum.GetValue()
+		}
+	}
```

Comment:
Let's cover this logic with a test please

Reply:
https://github.com/osmosis-labs/osmosis/pull/4265/commits/9e749648ad3bb9d1b9feb6c5ffb5207cef1dcb75